frontend/src/components/dashboard/aliases/MaskCard.module.scss

```
@@ -345,19 +354,20 @@
       }
 
       &.is-disabled {
-        cursor: help;
         padding-inline: $spacing-md;
-        background-color: transparent;
         color: $color-grey-30;
-        text-decoration: line-through;
 
-        &:hover {
-          color: $color-red-70;
+        @media screen and #{$mq-md} {
+          svg {
+            flex: 1 0 10px;
+          }
```

I think if you don't set `flex-shrink` to 0, you don't need the media query - unless I'm missing the reason for it?

```suggestion
        svg {
          flex: 1 1 10px;
        }
```